packages/AlgernonTests.package/ALGTestAction.class/instance/setUp.st

```diff
@@ -1,4 +1,3 @@
 running
 setUp
-	actions := ALGAction createListOfAlgeractionsGiven: (ALGClassType with: ALGCore).
-	self core: ALGCore new. 
```

Comment:
I'm not 100% sure with this one, but from my testing removing this line doesn't alter the behavior. I modified a method in Algernon in such a way that the assertion fails, and it also did without this line. Plus removing this line causes the screen not to go white anymore when the tests in ALGTestAction are run.

Reply:
Seems good!